src/Apps/Sell/Routes/AdditionalRoutes/ShippingLocationRoute.tsx

```diff
@@ -1,114 +1,230 @@
 import { SubmissionLayout } from "Apps/Sell/Components/SubmissionLayout"
 import { SubmissionStepTitle } from "Apps/Sell/Components/SubmissionStepTitle"
 import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
-import {
-  LocationAutocompleteInput,
-  Place,
-  buildLocationDisplay,
-  normalizePlace,
-} from "Components/LocationAutocompleteInput"
+import { CountrySelect } from "Components/CountrySelect"
+import { extractNodes } from "Utils/extractNodes"
+import { ShippingLocationRoute_me$key } from "__generated__/ShippingLocationRoute_me.graphql"
 import { ShippingLocationRoute_submission$key } from "__generated__/ShippingLocationRoute_submission.graphql"
-import { Formik } from "formik"
+import { Form, Formik } from "formik"
+import { last } from "lodash"
 import * as React from "react"
 import { graphql, useFragment } from "react-relay"
 import * as Yup from "yup"
 
-const FRAGMENT = graphql`
+const SUBMISSION_FRAGMENT = graphql`
   fragment ShippingLocationRoute_submission on ConsignmentSubmission {
     locationCity
     locationCountry
-    locationPostalCode
     locationState
+    locationPostalCode
+    locationAddress
+    locationAddress2
+  }
+`
+
+const ME_FRAGMENT = graphql`
+  fragment ShippingLocationRoute_me on Me {
+    addressConnection {
+      edges {
+        node {
+          addressLine1
+          addressLine2
+          city
+          country
+          isDefault
+          postalCode
+          region
+        }
+      }
+    }
   }
 `
 
 const Schema = Yup.object().shape({
   location: Yup.object().shape({
-    locationCity: Yup.string().trim(),
-    locationState: Yup.string(),
-    stateCode: Yup.string(),
-    locationPostalCode: Yup.string(),
-    locationCountry: Yup.string(),
+    city: Yup.string().required().trim(),
+    state: Yup.string().required().trim(),
+    country: Yup.string().required().trim(),
+    zipCode: Yup.string().required().trim(),
+    address: Yup.string().required().trim(),
+    address2: Yup.string().trim(),
   }),
 })
 
 interface FormValues {
   location: {
-    city: string
-    state: string
-    stateCode: string
-    country: string
+    city?: string
+    state?: string
+    country?: string
+    zipCode?: string
+    address?: string
+    address2?: string
   }
 }
 
 interface ShippingLocationRouteProps {
+  me: ShippingLocationRoute_me$key
   submission: ShippingLocationRoute_submission$key
 }
 
 export const ShippingLocationRoute: React.FC<ShippingLocationRouteProps> = props => {
-  const submission = useFragment(FRAGMENT, props.submission)
   const { actions } = useSellFlowContext()
 
+  const submission = useFragment(SUBMISSION_FRAGMENT, props.submission)
+  const me = useFragment(ME_FRAGMENT, props.me)
+
+  const userAddresses = extractNodes(me?.addressConnection)
+  const defaultUserAddress =
+    userAddresses.find(node => node.isDefault) || last(userAddresses || [])
+
+  // Check whether the submission has an address; if not, use the user's address.
```

Comment:
I think I forgot to query for the user default address in app - I can add that too

Reply:
Good idea 👍 Maybe you can reuse some of the code from this PR in the app.